test/MOL/MOL_1D_Linear_Diffusion.jl

```diff
@@ -265,159 +263,15 @@ end
 
     # Solve ODE problem
     sol = solve(prob,Tsit5(),saveat=0.1)
-    x = dx[2:end-1]
-    t = sol.t
-
-    # Test against exact solution
-    for i in 1:length(sol)
-        exact = u_exact(x, t[i])
-        u_approx = sol.u[i]
-        @test u_approx ≈ exact atol=0.01
-    end
-end
-
-@testset "Test 06: Dt(u(t,x)) ~ Dxx(u(t,x)), time-dependent Robin BCs" begin
-    # Method of Manufactured Solutions
-    u_exact = (x,t) -> exp.(-t) * sin.(x)
-
-    # Parameters, variables, and derivatives
-    @parameters t x
-    @variables u(..)
-    Dt = Differential(t)
-    Dx = Differential(x)
-    Dxx = Differential(x)^2
-
-    # 1D PDE and boundary conditions
-    eq  = Dt(u(t,x)) ~ Dxx(u(t,x))
-    bcs = [u(0,x) ~ sin(x),
-           t^2 * u(t,-1.0) + 3Dx(u(t,-1.0)) ~ exp(-t) * (t^2 * sin(-1.0) + 3cos(-1.0)),
-           4u(t,1.0) + t * Dx(u(t,1.0)) ~ exp(-t) * (4sin(1.0) + t * cos(1.0))]
-
-    # Space and time domains
-    domains = [t ∈ IntervalDomain(0.0,1.0),
-               x ∈ IntervalDomain(-1.0,1.0)]
-
-    # PDE system
-    pdesys = PDESystem(eq,bcs,domains,[t,x],[u(t,x)])
-
-    # Method of lines discretization
-    dx = 0.01
-    order = 2
-    discretization = MOLFiniteDifference([x=>dx],t)
-
-    # Convert the PDE problem into an ODE problem
-    prob = discretize(pdesys,discretization)
-
-    # Solve ODE problem
-    sol = solve(prob,Tsit5(),saveat=0.1)
-    x = prob.space[2]
-    t = sol.t
-
-    x = dx[2:end-1]
+    x = (-1:dx:1)[2:end-1]
```

Comment:
With the deletions we should uncomment this test now?

Reply:
Yes this test file now works. Except test 6 which creates a DAE system (I guess it could work with a DAE solver, didn't try), and test 2 with the nonlinear laplacian

Reply:
I'll take a quick look.